From 09733224067f465c956ff2173c46ac43c6ec4a9c Mon Sep 17 00:00:00 2001
From: Jiri Olsa <jolsa@redhat.com>
Date: Mon, 8 Sep 2014 13:08:20 -0400
Subject: [tools] perf: Enable --children option by default

Message-id: <1410181752-25631-256-git-send-email-jolsa@redhat.com>
Patchwork-id: 91938
O-Subject: [PATCH RHEL7.1 BZ1134356 255/307] perf tools: Enable --children option by default
Bugzilla: 1134356
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Don Zickus <dzickus@redhat.com>

From: Jiri Olsa <jolsa@kernel.org>

Bugzilla: 1134356
https://bugzilla.redhat.com/show_bug.cgi?id=1134356

upstream
========
commit e511db5e94f056083e821aa3ab74b03ad1216e14
Author: Namhyung Kim <namhyung@kernel.org>
Date: Tue Dec 24 16:19:25 2013 +0900

description
===========
Now perf top and perf report will show children column by default if
it has callchain information.
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/tools/perf/util/symbol.c b/tools/perf/util/symbol.c
index 95e2497..7b9096f 100644
--- a/tools/perf/util/symbol.c
+++ b/tools/perf/util/symbol.c
@@ -29,11 +29,12 @@ int vmlinux_path__nr_entries;
 char **vmlinux_path;
 
 struct symbol_conf symbol_conf = {
- .use_modules   = true,
- .try_vmlinux_path = true,
- .annotate_src   = true,
- .demangle   = true,
- .symfs            = "",
+ .use_modules  = true,
+ .try_vmlinux_path = true,
+ .annotate_src  = true,
+ .demangle  = true,
+ .cumulate_callchain = true,
+ .symfs   = "",
 };
 
 static enum dso_binary_type binary_type_symtab[] = {
-- 
1.7.1